py-polars/pyproject.toml

```diff
@@ -164,6 +164,7 @@ strict = true
 
 [tool.ruff.per-file-ignores]
 "tests/**/*.py" = ["D100", "D103", "B018"]
+"polars/type_aliases.py" = ["D101", "D102"]
```

Comment:
Please just use `# noqa` comments for these.

Or even better, write a small docstring.

Reply:
Lol... done  ;)